src/knex.ts

```diff
@@ -0,0 +1,16 @@
+import { knex as _knex, Knex } from 'knex'
+import env from './env'
+
+let instance: Knex | undefined
+export default function knex() {
+	if (!env.DATABASE_URI) {
+		throw new Error('DATABASE_URI variable not found')
+	}
+	if (instance === undefined) {
+		instance = _knex({
+			client: 'pg',
+			connection: env.DATABASE_URI
+		})
+	}
+	return instance
+}
```

Comment:
```suggestion
let instance: Knex | undefined

const {knex} = new class {
	get knex() {
		if (!env.DATABASE_URI) {
		throw new Error('DATABASE_URI variable not found')
	}
	if (instance === undefined) {
		instance = _knex({
			client: 'pg',
			connection: env.DATABASE_URI
		})
	}
	return instance
	}
}

export default knex
```

Reply:
this approach doesn't crash for importing, but using it in a CommandHandler function crashes the app on launch.

Reply:
Can I see an example handler that crashes?